source/_docs/z-wave/installation.markdown

```diff
@@ -53,7 +53,7 @@ usb_path:
   type: string
   default: /zwaveusbstick
 network_key:
-  description: The 16-byte network key in the form `"0x01, 0x02..."` used in order to connect securely to compatible devices.
+  description: The 16-byte network key in the form `"0x01, 0x02..."` used in order to connect securely to compatible devices. It is recommended that a network key is configured as security enabled devices will not function correctly if they are not added securely.
```

Comment:
Given that some/most devices that support Secure function just fine without it, maybe "_may not function correctly_" instead?

Reply:
Oh do they?  I always thought that they would pair but certain parts of functionality would not work.  I was not aware that they will function 100% without a secure connection.